src/dependencies/i18n.messages.ts

```diff
@@ -38,6 +39,7 @@ const messages = {
             MIN_LENGTH_AMOUNT: 'Dit veld moet minimaal {length} karakters bevatten',
             NO_ZERO: 'Mag niet nul zijn',
             VALID_CURRENCY: 'Gebruik geldige valutanotatie',
+            MAX_NUMBER_AMOUNT: 'Max {size}',
```

Comment:
there is a different ket for the two translations `MAX_LENGTH_AMOUNT` and `MAX_NUMBER_AMOUNT`

Reply:
Correct, I did rename the property, forgot to rename in NL